src/main/java/pacman/ActionKeyPacman.java

```diff
@@ -7,8 +7,14 @@
 public class ActionKeyPacman implements KeyListener {
 
     private DrawComponents drawComponents;
-    private int key;
     public static final int TIMER_PACMAN = 200;
+    public static final String MOVE_LEFT = "left";
+    public static final String MOVE_RIGHT = "right";
+    public static final String MOVE_UP = "up";
+    public static final String MOVE_DOWN = "down";
+    private int key, preKey;
+    private Boolean aux = true;
```

Comment:
maybe you could change the name of the "aux" variable to a more descriptive one

Reply:
thank you, it's changed in the last commit